src/main/scala/com/fortysevendeg/android/sqlite/statement/SQLDroidStatement.scala

```diff
@@ -0,0 +1,198 @@
+package com.fortysevendeg.android.sqlite.statement
+
+import java.sql._
+
+import com.fortysevendeg.android.sqlite.logging.{AndroidLogWrapper, LogWrapper}
+import com.fortysevendeg.android.sqlite.resultset.SQLDroidResultSet
+import com.fortysevendeg.android.sqlite.{SQLDroidConnection, SQLDroidDatabase, WrapperNotSupported}
+
+import com.fortysevendeg.android.sqlite._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/811484953/issues/source?bid=2442814&fileBranchId=2740117#l9)